amethyst_renderer/src/pass/pbm/interleaved.rs

```diff
@@ -94,18 +94,10 @@ where
     fn apply<'a, 'b: 'a>(
         &'a mut self,
         supplier: Supplier<'a>,
-        (
-            camera,
-            ambient,
-            mesh_storage,
-            tex_storage,
-            material_defaults,
-            mesh,
-            material,
-            global,
-            light,
-        ): (
-            Option<Fetch<'a, Camera>>,
+        (active, camera, ambient, mesh_storage, tex_storage, material_defaults,
```

Comment:
I'm not sure about this formatting, but well

Reply:
That formatting was merged in from @Rhuagh 's PR, but I still have more code to write here so I'll give it a rustfmt pass again.

Reply:
That's weird. It shouldn't do that